pkgs/development/python-modules/runstats/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  setuptools,
+  cython,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "runstats";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "grantjenks";
+    repo = "python-runstats";
+    rev = "v${version}";
+    hash = "sha256-YF6S5w/ccWM08nl9inWGbaLKJ8/ivW6c7A9Ny20fldU=";
+  };
+
+  build-system = [
+    setuptools
+    cython
+  ];
+
+  pythonImportsCheck = [ "runstats" ];
+
+  meta = with lib; {
```

Comment:
remove with lib

Reply:
I thought this was the preferred way to add meta field looking through all the other python packages. But i changed it